siteimprove/admin/js/siteimprove-gutenberg.js

```diff
@@ -25,8 +25,14 @@ var RecheckButton = function () {
         },
         onClick: function () {
             setClicked(true);
-            if (typeof window.siteimprove !== "undefined" && typeof window.siteimprove.recheck !== "undefined") {
-                window.siteimprove.recheck(siteimprove_gutenberg_recheck.url, siteimprove_gutenberg_recheck.token, function() { setClicked(false) });
+            if (typeof window.siteimprove !== "undefined") {
```

Comment:
This is not the part of the if that was supposed to be enough, it was typeof window.siteimprove.recheck !== "undefined"